docs/public-networks/how-to/connect/manage-peers.md

```diff
@@ -31,10 +31,11 @@ To reduce the maximum number of peers, use the [`--max-peers`](../../reference/c
 
 The minimum number of peers is set by the `--Xp2p-peer-lower-bound` option, which also has a default of 25. If you reduce the `--max-peers` from the default, you must also set the `--Xp2p-peer-lower-bound` option to the same value or lower. For example, if you decrease `--max-peers` to 20, set `--Xp2p-peer-lower-bound` to 20 or lower.
 
-Note, `Xp2p-peer-lower-bound` is an early access option.
+`--Xp2p-peer-lower-bound` is an early access option.
 
 :::
 
+
```

Comment:
```suggestion
```

Please always check your "files changed" or run `git diff` before staging to make sure you're not committing accidental changes such as unnecessary line breaks.

Reply:
I edited the PR and thought I made this change. Just an error on my end